.github/workflows/release.yml

```diff
@@ -40,17 +40,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Download
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
-          name: packages
+          path: packages
+          pattern: package-*
```

Comment:
Oh, actually, I have a question: Will the [path the Autobuilder uses](https://github.com/mongodb/docs-worker-pool/blob/24ec3e5c0a4b4f6c6a421862680d095063d4f816/Dockerfile#L51) to download the packages need to be updated?

Reply:
Should just work! The zip file names aren't changing, and they look to be what I expect in the test pseudo-release I did*.

* which is slightly different from a real release, but only in a way that *shouldn't* matter.